src/include/mallocMC/creationPolicies/Scatter_impl.hpp

```diff
@@ -132,6 +132,8 @@ namespace ScatterKernelDetail{
       //static const uint32 minChunkSize0 = pagesize/(32*32);
       static const uint32 minChunkSize1 = 0x10;
       static const uint32 HierarchyThreshold =  (pagesize - 2*sizeof(uint32))/33;
+      static const uint32 tmp_maxOPM = minChunkSize1 > HierarchyThreshold ? 0 : pagesize/(32*minChunkSize1 + 1);
```

Comment:
Thanks for the fast reply.
Is it possible to create a `static const` variable for `(32*minChunkSize1 + sizeof(uint32))` to avoid mistakes?